src/notebook/notebookFeature.ts

```diff
@@ -342,8 +335,17 @@ export class JuliaNotebookFeature {
             },
         }
 
-        if (this.vscodeIpynbApi && !isEqual(getNotebookMetadata(notebook), metadata)) {
-            this.vscodeIpynbApi.setNotebookMetadata(notebook.uri, metadata)
+        if (!isEqual(getNotebookMetadata(notebook), metadata)) {
+            const edit = new vscode.WorkspaceEdit();
```

Comment:
This is the exact same code from this PR https://github.com/microsoft/vscode/pull/132298/files